src/advanced/bitcoin-anchoring.yaml

```diff
@@ -0,0 +1,259 @@
+
+servers:
+- url: http://example.com/api/services/btc_anchoring/v1
+
+tags:
+- name: Read Requests
+  description: Public read-only API available to everybody.
+
+paths:
+  /address/actual:
+    get:
+      tags: [Read Requests]
+      summary: Get actual anchoring address
+      description: >
+        Returns the current anchoring BTC-address.
+      responses:
+        200:
+          description: >
+            The string with a value of the anchoring address in the Base58Check format.
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/BtcAddress"
+
+  /address/following:
+    get:
+      tags: [Read Requests]
+      summary: Get next anchoring address
+      description: >
+        If the change of the validators list is scheduled, returns the next anchoring address.
+        Otherwise, returns `null`.
+      responses:
+        200:
+          description: >
+            The string with a value of the anchoring address in the Base58Check format.
+          content:
+            application/json:
+              schema:
+                allOf:
+                  - $ref: "#/components/schemas/BtcAddress"
+                  - nullable: true
+
+  /actual_lect:
+    get:
+      tags: [Read Requests]
+      summary: Get actual common LECT
+      description: >
+        Returns the [LECT](https://exonum.com/doc/advanced/bitcoin-anchoring/#lect)
+        that is agreed upon by the supermajority of validators now, if such exists.
+        Otherwise, returns `null`.
+      responses:
+        200:
+          description: >
+            JSON object with anchoring information or `null`.
+          content:
+            application/json:
+              schema:
+                allOf:
+                  - $ref: "#/components/schemas/AnchoringInfo"
+                  - nullable: true
+              example:
+                txid: "021dd89bd3343a8a6ad259fbe1eed638217358b262db66a9619af2ca92fb89d9"
+                payload:
+                  block_hash: "03c5d221357d5d10c20792d480ba29267f3895575fbe36bef175abab9e9c9f5a"
+                  block_height: 0
+                  prev_tx_chain: null
+
+  /actual_lect/validator:
+    get:
+      tags: [Read Requests]
+      summary: Get actual LECT for specific validator
+      description: >
+        Returns the actual [LECT](https://exonum.com/doc/advanced/bitcoin-anchoring/#lect)
+        for the specified validator,
+        along with the hash of the Exonum transaction published in this LECT.
+
+        If the specified validator `id` is greater or equal to the total validators amount
+        then an error is returned.
+      parameters:
+        - in: query
+          name: id
+          required: true
+          description: Identifier of the validator to query for.
+          schema:
+            type: integer
+            minimum: 0
+            maximum: 4294967295
```

Comment:
[Validator is represented by `u16`](https://docs.rs/exonum/0.8.0/exonum/helpers/struct.ValidatorId.html), so the upper limit should be 65535.